lib/documents/schemas/asylum_support_decisions.json

```diff
@@ -111,6 +111,10 @@
           "value": "section-4-2-jurisdiction"
         },
         {
+          "label": "Section 4(2) - other",
+          "value": "section-4-2-other"
+        },
+        {
           "label": "Section 4(2) - regulation 3(2) - other",
```

Comment:
Should the additions replace "Section 4(2) - regulation 3(2) - other", or are they fine to just be extra?